.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-- 12
-- 14
+- 16
```

Comment:
16 is going to EOL in 32 days, I'd add 18 too here, if it passes without extra work

![image](https://github.com/mapbox/quilt/assets/2198527/22fdc975-3c25-41d5-9196-43eb035798f0)